Vsts-Extension/Set-AzureAdApplication/Set-AzureAdApplicationV2/task.json

```diff
@@ -233,8 +233,8 @@
             "description": "The ObjectId of the Service Principal connected to the Azure AD Application"
         },
         {
-            "name": "Secret.$Description",
-            "description": "The secret generated for the Azure AD Application. If multiple secrets are requested, an output variable for each secret will be available using the convention .Secret.$Description (where $Description is equal to the Description provided in the 'Secrets' input field)"
+            "name": "Secret.MyDescription",
+            "description": "The secret generated for the Azure AD Application. If multiple secrets are requested, an output variable for each secret will be available using the convention .Secret.[Description] (where [Description] is equal to the Description provided in the 'Secrets' input field)"
```

Comment:
using the convention: Secret.[Description] (where [Description] is equal to the Description provided in the 'Secrets' input field)

the . before Secret is unclear. Use convention semi colon? So:
convention: Secret.[Description]